test/es6/object_literal_bug.js

```diff
@@ -1,43 +1,56 @@
-    body: function () {
-        (function () {
-            let a = 1;
-            {
-                let a = 2;
-                var m = {a};
-                assert.areEqual(m.a, 2, "Name node in object literal shorthand is binding correctly with inner block");
-            }
-        })();
-
-        {
-            let a2 = 1;
-            ({a2} = {a2:2});
-            assert.areEqual(a2, 2, "Destructuring object pattern : Name node in object literal shorthand is binding correctly with inner block");
-        }
-        {
-            // Object literal shorthand - referenced in different scopoe works correctly.
-            { d; }
-
-            {
-                { { d;} };
-                var c = {d};
-            }
-
-            {
-                var d = [];
-            }
-        }
-    }
-  }
-];
-
-testRunner.runTests(tests, { verbose: WScript.Arguments[0] != "summary" });
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+  {
+    name: "Object literal shorthand syntax with symbol lookup",
+    body: function () {
+        (function () {
+            let a = 1;
+            {
+                let a = 2;
+                var m = {a};
+                assert.areEqual(m.a, 2, "Name node in object literal shorthand is binding correctly with inner block");
+            }
+        })();
+
+        {
+            let a2 = 1;
+            ({a2} = {a2:2});
+            assert.areEqual(a2, 2, "Destructuring object pattern : Name node in object literal shorthand is binding correctly with inner block");
+        }
+        {
+            // Object literal shorthand - referenced in different scopoe works correctly.
+            { d; }
+
+            {
+                { { d;} };
+                var c = {d};
+            }
+
+            {
+                var d = [];
+            }
+        }
+    }
+  },
+  {
+    name: "Getter/setter methods defined in object literal should not have 'prototype' property",
+    body: function () {
+        var o = {
+            get m() {},
+            set m(v) {}
+        };
+        var g = Object.getOwnPropertyDescriptor(o, 'm').get;
+        var s = Object.getOwnPropertyDescriptor(o, 'm').set;
+        assert.areEqual(false, g.hasOwnProperty('prototype'), "Getter method defined in object literal should not have 'prototype' property");
+        assert.areEqual(false, s.hasOwnProperty('prototype'), "Setter method defined in object literal should not have 'prototype' property");
+    }
+  },
+];
+
+testRunner.runTests(tests, { verbose: WScript.Arguments[0] != "summary" });

```

Comment:
Thanks to @dilijev for clarifying. Let me recap in case I misunderstood:
1) Jenkins EOL check doesn't operate on .js files because there are cases that are intentional
2) This file: test/es6/object_literal_bug.js , had Unix EOL (\n) and was changed to Windows EOL (\r\n) by git when it checked out. This is expected with autocrlf==true
3) When git checks in this file, it didn't complain because we have specifically set it such that .js files' EOL shall be left alone (same reason as 1)
4) Given 2) and 3), it would've helped reducing the size of this commit if I had explicitly run a 'dos2unix' on this file.
5) Sampling of two other .js files showed they have Windows EOL. There may be files with Unix EOL out there but we hadn't seen it. So far they are not causing any issues.
6) There's no compelling value to change test/es6/object_literal_bug.js back to Unix EOL. It might have become more consistent with other .js files in the same directory.

Therefore the recommended course of action is not to change back to Unix EOL for test/es6/object_literal_bug.js

@ianwjhalliday thanks for asking good questions. Let me know if you have other questions or concerns.